apps/wq_hypersweep/test.py

```diff
@@ -0,0 +1,64 @@
+from work_queue import *
+import sys
+
+
+def compose_task(i,j):
+	id = (i-1)*20+j
+	d_rate = i*0.05
+	r_blok = j
+	outfile = "results%d.csv" % id
+	command = "./script.sh results%d.csv %f %d" % (id,d_rate,r_blok)
+
+	t = Task(command)
+
+	t.specify_file("env.tar.gz", "env.tar.gz", WORK_QUEUE_INPUT, cache=False)
```

Comment:
I think all these inputs should be cacheable since they don't change throughout the run, yes?